.github/workflows/integration-tests.yml

```diff
@@ -2,47 +2,84 @@ name: Integration tests
 
 on:
   workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to run the tests against'
+        type: choice
+        required: true
+        default: 'dev'
+        options:
+          - dev
+          - staging
   workflow_call:
+    inputs:
+      environment:
+        default: 'staging'
+        required: false
+        type: string
     secrets:
-      FIREBOLT_USERNAME:
-        required: true
-      FIREBOLT_PASSWORD:
-        required: true
+      FIREBOLT_USERNAME_STAGING:
+        required: false
+      FIREBOLT_PASSWORD_STAGING:
+        required: false
+      FIREBOLT_CLIENT_ID_STAGING:
+        required: false
+      FIREBOLT_CLIENT_SECRET_STAGING:
+        required: false
+      FIREBOLT_USERNAME_DEV:
+        required: false
+      FIREBOLT_PASSWORD_DEV:
+        required: false
+      FIREBOLT_CLIENT_ID_DEV:
+        required: false
+      FIREBOLT_CLIENT_SECRET_DEV:
+        required: false
 
 jobs:
   integration-tests:
     runs-on: ubuntu-latest
     steps:
       - name: Check out code
-        uses: actions/checkout@v2
-
+        uses: actions/checkout@v3
```

Comment:
Any differences between v2 and v3?

Reply:
v2 is being deprecated (so there are warning poluting the GHA summary) 🤢

Reply:
We should do this for all of our repos. Could you file a ticket to track it please?